src/style.css

```diff
@@ -1,3 +1,88 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap';
+
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  position: relative;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  background-color: #fff;
+}
+
+.nav {
+  display: flex;
+  margin: 0 50px 0 0;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+ul :first-child a {
+  margin: 0 10px 0 0;
+}
+
+ul :last-child a {
+  margin: 0 0 0 10px;
+}
+
+.nav__link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 60px;
+  position: relative;
+  margin: 0 10px;
+  color: #000;
+  text-decoration: none;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.nav__logo {
+  position: absolute;
```

Comment:
Using position: absolute for logo can lead to layout issues so instead it should be positioned within the header for better alignment and responsiveness. This approach ensures that logo stays properly positioned relative to other elements